spec/factories/user_profiles.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :user_profile do
```

Comment:
Shouldn't the factory definition require a `user`? Since we already have a `user` factory, you could add something like the following:

```ruby
FactoryGirl.define do
  factory :user_profile do
    user
  end
end
```

What do you think @sejalkhatri ?